sass/components/html/file--link.html

```diff
@@ -1,8 +1,11 @@
 <div class="file file--link">
-  <span class="file__icon icon icon--file-link"></span>
```

Comment:
Why did you remove/change this?
We need to understand if this is feasible because I think there is a strong dependency with the file component and his drupal implementation.
cc: @planctus